### DISCUSSION TITLE
Custom Pages for Tags

### DISCUSSION THREAD

**Discussion by antforager** [bootscore/bootscore#872] [Q&A]
For my ant keeping pet store, I have an "Ants" category. Some of the ants have a "beginner-ants" tag which indicates they are easy to care for.  I want to have a specific page for Beginner Ants. It will have a "Beginner Ants" title, a paragraph explaining why the ants are good for new ant keepers, and then list the products with the beginner-ants tag in the same format as the default category page. 

All the ants are listed here: https://6b5.042.myftpupload.com/product-category/ants/
The beginner-ants tag would be here: https://6b5.042.myftpupload.com/product-tag/beginner-ants/

I have been trying to figure out where to put a template file called taxonomy-product_tag-beginner-ants.php. I have tried putting it in the root directory of my child theme, inside a "woocommerce" directory, and "/woocommerce/templates" but it isn't showing my changes. 

Where would I put the file? 

Is there a better way to do this? It's going to be repetitive because I have 3 or 4 other tags to make pages for. Can I have a filter update the description part?

Thanks,
Cindy

**Comment by crftwrk** [comment]
> I have been trying to figure out where to put a template file called taxonomy-product_tag-beginner-ants.php

This won't work because is no way to add a different template for tags. Also it is not possible to create a custom template for a product category because the theme uses `woocommerce.php` that does not allow overwriting the `archive-product.php`. See https://developer.woocommerce.com/docs/template-structure-overriding-templates-via-a-theme/#4-declare-theme-support-for-custom-templates

Two workarounds:

#### 1. Use category instead tags

When using categories for sorting instead tags, you can simply add a category description. HTML is allowed here, so you can add to the description whatever you want. 

#### 2. Use WC shortcodes

Create a new page and add a WC shortcode to list products by tags:

`[products tag="beginner-ants"]`

Design the page however you want. See https://woocommerce.com/document/woocommerce-shortcodes/products/#display-product-attributes

Is this helpful?
